src/instructlab/server.py

```diff
@@ -71,7 +71,7 @@ def ensure_server(
         return (None, None, None)
     except ClientException:
         tried_ports = set()
-        mpctx = multiprocessing.get_context(None)
+        mpctx = multiprocessing.get_context(DEFAULT_MULTIPROCESSING_START_METHOD)
```

Comment:
Are you sure this changes behavior?  When passing `None`, it should use the currently set default context. In other words, I don't think this change does anything.  Can you let me know if I'm wrong?

Reply:
That's correct `None` will point to the default context. On Darwin, it's `spawn`, on Linux it's `fork`. We want to force `spawn` on all platforms.

@tiran please correct me if I'm wrong.

Reply:
Multiprocessing has two different types of defaults. There is the initial default setting for the context and there is the global default context. On Linux, the initial default is `fork` for Python <= 3.13. Python 3.14 will use spawn. We force-set `spawn` everywhere and override the value for `get_context(None)`:

```python
>>> import multiprocessing
>>> multiprocessing.get_context(None)
<multiprocessing.context.ForkContext object at 0x7f40a5b4bf50>
>>> multiprocessing.set_start_method("spawn", force=True)
>>> multiprocessing.get_context(None)
<multiprocessing.context.SpawnContext object at 0x7f40a5b4bfe0>
```

Reply:
Oh ok so this is the override:

https://github.com/instructlab/instructlab/blob/167ccd81f83dafbeb58d5f48fa2e87b2b7d45a6d/src/instructlab/lab.py#L31-L33

I'm closing this PR then. Thanks all!